pages/account/how-to/use-2fa.mdx

```diff
@@ -3,98 +3,117 @@ title: How to use Multifactor Authentication (MFA)
 
-## How to update MFA
+### How to update TOTP
 
-If you no longer have access to the device in which you set up your MFA, you can update your MFA information and set it up via the Scaleway console.
+If you no longer have access to the device in which you set up your TOTP, you can update your TOTP information and set it up via the Scaleway console.
 
 <Requirements />
 
-- A Scaleway account logged into the [console](https://console.scaleway.com)
 - [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
-- [Enabled MFA](#how-to-enable-mfa) on your account
+- [Enabled TOTP](#how-to-enable-totp) on your account
 
 <Message type="important">
   If you have lost access to your account and are not able to log in, follow the [Cannot log into my account](/account/troubleshooting/cannot-log-into-my-account/) troubleshooting procedure.
 </Message>
 
 1. Go to your [Account Security](https://console.scaleway.com/settings/account/security) page.
-2. Click **Update MFA**, in the **Multifactor authentication** section. A pop-up displays.
+2. Click **Update TOTP**, in the **Multifactor authentication** section. A pop-up displays.
 3. Enter the code shown on the pop-up into your MFA app, or scan the QR code into your app.
-  Your app sets up MFA for your Scaleway account and displays a 6-digit code.
-4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**. A pop-up displays.
-5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way to regain access to your account without your MFA app.
+  Your app sets up TOTP for your Scaleway account and displays a 6-digit code.
+4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**.
+5. Download or copy the backup codes shown, and store them in a safe place. These codes will be the only way to regain access to your account without your TOTP app.
+
+### How to delete TOTP
+
+1. Go to your [Account Security](https://console.scaleway.com/settings/account/security) page.
+2. Click **Delete TOTP**, in the **Multifactor authentication** section. A pop-up displays.
+3. Type **DELETE** in the box to confirm, then click **Delete TOTP**.
 
-## How to disable MFA
+TOTP is now disabled on your account.
 
 <Message type="important">
-  If you disable MFA as a Member of an Organization that has MFA enforced, you will need to re-enable it before the Organization's [grace period](/iam/how-to/comply-with-sec-requirements-member/#grace-period) ends, otherwise your Member account will be locked.
+  If you delete TOTP as a Member of an Organization that has MFA enforced, you will need to re-enable it before the Organization's [grace period](/iam/how-to/comply-with-sec-requirements-member/#grace-period) ends, otherwise your Member account will be locked.
```

Comment:
this is not if you delete totp anymore, this occurs when you delete your last mfa method (if you have a passkey and totp, you can delete totp without disabling mfa thus not triggering this use case)